content/projects/gsoc/gsoc2018-project-ideas.adoc

```diff
@@ -143,6 +146,36 @@ Potential mentors:
 link:https://github.com/christ66[Steven Christou],
 link:https://github.com/Jeff-Symphony[Jeff Knurek]
 
+=== Simple Pull-Request Job Plugin
```

Comment:
This can be done with Declarative Pipeline almost right out of the box.  If they have to use Job DSL to create or configure it, they might as well be writing a `Jenkinsfile`.

Reply:
Yes, there is a long discussion thread in the Google doc. I rather anticipate it to be a number of enhancements in existing plugns